src/util/MongoInstance.ts

```diff
@@ -54,7 +54,14 @@ export default class MongodbInstance {
       this.opts.binary.debug = this.opts.debug;
     }
 
-    if (this.opts.instance && this.opts.instance.debug) {
+
+    this.debug = (msg: string) : void => {
```

Comment:
@nodkz Would this be enough to handle the debug function in the MongoInstance class ? 
With the multipleDB-test suite, you can see that each instance debug log is prefixed with the right `Mongo[${port}]` value.

Reply:
For now, I think yes.

In future it can be easily improved ;)